var/spack/repos/builtin/packages/root/package.py

```diff
@@ -248,6 +248,10 @@ class Root(CMakePackage):
     depends_on('shadow',    when='+shadow')
     depends_on('sqlite',    when='+sqlite')
     depends_on('tbb',       when='+tbb')
+    # See: https://github.com/root-project/root/issues/6933
+    conflicts('^intel-tbb@2021.1:', when='@:6.22',
+              msg='Please use an older intel-tbb version')
+    # depends_on('intel-tbb@:2021.0', when='@:6.22 ^intel-tbb')
```

Comment:
Is  there a reason to keep this comment around?

Reply:
Well, it describes much better, what we really want. But it does not work well with the classic concretizer. Once we have the new as the default concretizer, I would like to switch to this one.

That said: Do you want me to remove it?

Reply:
Agreed that this is the most  elegant way to deal with the issue, and it is good to have it here for the future. Maybe just add a comment with a little explanation?